.github/workflows/e2e_spark.yaml

```diff
@@ -0,0 +1,68 @@
+name: E2E Spark Integration Test
+
+on:
+  # Post check in below branches
+  push:
+    branches:
+      - master
+  # PR triggered job
+  pull_request:
+jobs:
+  k8s-integration-tests:
+    name: "Run Spark (branch-3.3) / Volcano Integration test"
+    runs-on: ubuntu-20.04-spark
+    steps:
+
+    - name: Checkout current Volcano repository
+      if: github.event.inputs.volcano-branch==''
+      uses: actions/checkout@v2
+
+    - name: Checkout Spark repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        repository: apache/spark
+        ref: branch-3.3
+        path: ${{ github.workspace }}/spark
+    - name: Install Java 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
+    - name: start minikube
+      run: |
+        # Use pre-install minikube
+        minikube start
```

Comment:
Can we us `kind`? In my impression, volcano uses `kind` to test more often.

Reply:
In theory, kind and minikube have similar functions. Spark are using `minikube`, there are also some related advanced test depends on `minikube` (such as PV test), so it's better to keep same to use minikube if no more objection, to ensure we can enable more `minikube` related spark integration test in future.